src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/ContactInformation.tsx

```diff
@@ -45,11 +46,16 @@ export const ContactInformation: React.FC<ContactInformationProps> = ({
     removeSubmission,
   } = useSubmission(id)
 
+  const handleRecaptcha = (action: RecaptchaAction) =>
+    new Promise(resolve => recaptcha(action, resolve))
```

Comment:
**question:** How do we know whether a user passed or failed the reCAPTCHA test?

Reply:
recaptcha(action, **resolve**))

`resolve` function will receive the recaptcha token that as far as I understood indicates that the user passed.